include/hax_interface.h

```diff
@@ -269,4 +274,14 @@ struct hax_qemu_version {
     uint32_t least_version;
 } PACKED;
 
+#define HAX_DEBUG_ENABLE     (1 << 0)
+#define HAX_DEBUG_STEP       (1 << 1)
+#define HAX_DEBUG_USE_SW_BP  (1 << 2)
+#define HAX_DEBUG_USE_HW_BP  (1 << 3)
+
+struct hax_debug_t {
+    uint32_t control;
+    uint64_t dr[8];
+} PACKED;
```

Comment:
`PACKED` is ignored for Windows builds, so I suspect there's an implicit `uint32_t` inserted after `control`. How about we manually declare it as `uint32_t reserved`? It'll also give us room for extension.

Reply:
> How about we manually declare it as `uint32_t reserved`?

Agreed.